swap_meet/vendor.py

```diff
@@ -1,2 +1,94 @@
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self,inventory=None): 
+        self.inventory = [] if inventory is None else inventory
+        
+    def add(self, item_to_add):
+        self.inventory.append(item_to_add)
+        return item_to_add
+    
+    def remove(self, item_to_remove):
+        if item_to_remove not in self.inventory:
+            return False
+        else:
+            self.inventory.remove(item_to_remove)
+            return item_to_remove
+    
+    def get_by_id(self, item_id):
+        for item in self.inventory:
+            if item.id == item_id:
+                return item
+        return None
+    
+    def swap_items(self, other_vendor, my_item, their_item):  
+        if my_item in self.inventory and their_item in other_vendor.inventory:
```

Comment:
For checking if an item inside the inventory of a vendor we could, alternatively, use our `get_by_item` method. The method returns an item if it is found and `False` if it is not found. 
```python
        self_found = self.get_by_id(my_item.id)
        other_found = other_vendor.get_by_id(their_item.id)

        if self_found is None or other_found is None:
            return False
```